panc/src/main/java/org/quattor/pan/parser/PanParserAstUtils.java

```diff
@@ -285,8 +285,9 @@ static public Template convertAstToTemplate(File file, ASTTemplate ast) throws S
         // Create a list containing all of the statements.
         LinkedList<Statement> statements = new LinkedList<Statement>();
 
-        // By default the prefix for paths is empty.
-        Path prefix = null;
+        // Prefixes list: first element is the prefix, 2nd element the absolute prefix
```

Comment:
I completely mis-read this at first. Can you add the following to make it more obvious how this works?

2nd element the most recent absolute prefix, to which any new relative prefix is prepended.